src/composables/useMessageState.js

```
@@ -0,0 +1,149 @@
+          return;
+        }
+
+        if (currentIndex < fullText.length) {
+          const char = fullText[currentIndex];
+          const prevChar = currentIndex > 0 ? fullText[currentIndex - 1] : null;
+
+          currentText += char;
+
+          // Update the message in the SPECIFIC chat (not necessarily the active one)
+          const targetChat = allChats.get()[targetChatId];
+          if (targetChat && targetChat.messages[messageIndex]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)